drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/htmlunit/HtmlUnitDriver.java

```diff
@@ -0,0 +1,24 @@
+package org.jboss.arquillian.drone.webdriver.htmlunit;
+
+import com.gargoylesoftware.htmlunit.WebClient;
+import com.gargoylesoftware.htmlunit.WebClientOptions;
+import org.openqa.selenium.Capabilities;
+
+public class HtmlUnitDriver extends org.openqa.selenium.htmlunit.HtmlUnitDriver {
+
+    public HtmlUnitDriver(Capabilities capabilities) {
+        super(capabilities);
+    }
+
+    @Override
+    public WebClient modifyWebClient(WebClient client) {
+        final WebClientOptions webClientOptions = client.getOptions();
+
+        return super.modifyWebClient(client);
```

Comment:
Currently if we use HtmlDriver from selenium from which we couldn't get webClient. So we have to subclass it to get webclient & set it's options.  

About this method it's not doing anything because we already setup webClientOption [here](https://github.com/arquillian/arquillian-extension-drone/pull/85/files#diff-0771f250fc3a41a9424b87d1373b72a2R106). In future we may need to set some options e.g. CredentialsProvider which we can set when required.